### DISCUSSION TITLE
Some question about modification for our utilisation

### DISCUSSION THREAD

**Discussion by CM-AvenirData** [kimai/kimai#3422] [Questions & Answers]
Hi ! 

I have some question about the configuration of Kimai that i wanna make for utilisation.

1 - Can I set a maximum of hour in a week ? Like I wanna put 35h max hour, therefor, when my collaborators will register their timesheet and when they will cross the limit of 35h. They will cannot register their timesheet. 
Maybe can I modify the hour allowed in timesheet ? Maybe 9:00 to 12:30 and 13:30 to 17:00, this is possible ? 

2 - Can I remove Saturday and Sunday of the selection of day in timesheet ? 

3 - Can I create a recurrent projet to display like 'Eat Time ' in 12:30 to 13:30 ?

4 - Can I display Public Holiday in calendar and remove them from the selection of day ?

Ty for your future reply 🙏

**Comment by kevinpapst** [comment]
Kimai is a primarily a project-time tracker, not a time-tracker to manage "working time accounts".
You are looking for another type of software. In other words: 1,2,3 no and 4 partially
